app/helpers/render_constraints_helper_behavior.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+module RenderConstraintsHelperBehavior
+  include Blacklight::RenderConstraintsHelperBehavior
+
+  def remove_constraint_url(localized_params)
+    scope = localized_params.delete(:route_set) || self
+
+    unless localized_params.is_a? ActionController::Parameters
+      localized_params = ActionController::Parameters.new(localized_params)
+    end
+
+    options = localized_params.merge(q: nil, utf8: nil, action: article_search? ? 'new' : 'index')
```

Comment:
Hmm, is the only thing that we're overriding this method is to add the `action` ternary?

I'm wondering this is an opportunity to look into updating how we handle the article search home page to be more in-line w/ how Blacklight works (which could prevent the need to override any helpers at all).

Blacklight uses the index action to render both the search results  and the home page (using a helper to determine if there are query params that should be applied and a result rendered.

If we kill `article#new` and use the same helper, we may be able to render a `home_page` partial, and be more in-line w/ how the Blacklight helpers are expecting the UI to be organized.

(perhaps you already went down this path, but this gave me pause).